src/ert/config/queue_config.py

```diff
@@ -20,11 +19,53 @@
 
 GENERIC_QUEUE_OPTIONS: List[str] = ["MAX_RUNNING"]
 VALID_QUEUE_OPTIONS: Dict[Any, List[str]] = {
-    QueueSystem.TORQUE: _clib.torque_driver.TORQUE_DRIVER_OPTIONS
-    + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LOCAL: [] + GENERIC_QUEUE_OPTIONS,  # No specific options in driver
-    QueueSystem.SLURM: _clib.slurm_driver.SLURM_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
-    QueueSystem.LSF: _clib.lsf_driver.LSF_DRIVER_OPTIONS + GENERIC_QUEUE_OPTIONS,
+    QueueSystem.TORQUE: GENERIC_QUEUE_OPTIONS
+    + [
+        "CLUSTER_LABEL",
+        "DEBUG_OUTPUT",
+        "JOB_PREFIX_KEY",
+        "KEEP_QSUB_OUTPUT",
+        "MEMORY_PER_JOB",
+        "NUM_CPUS_PER_NODE",
+        "NUM_NODES",
+        "QDEL_CMD",
+        "QSTAT_CMD",
+        "QSTAT_OPTIONS",
+        "QSUB_CMD",
+        "QUEUE",
+        "QUEUE_QUERY_TIMEOUT",
+        "SUBMIT_SLEEP",
+    ],
+    QueueSystem.LOCAL: GENERIC_QUEUE_OPTIONS,
+    QueueSystem.SLURM: GENERIC_QUEUE_OPTIONS
+    + [
+        "SBATCH",
+        "SCONTROL",
+        "SQUEUE",
+        "SCANCEL",
+        "PARTITION",
+        "SQUEUE_TIMEOUT",
+        "MAX_RUNTIME",
+        "INCLUDE_HOST",
+        "EXCLUDE_HOST",
+    ],
```

Comment:
SLURM driver also currently has: `MEMORY` and `MEMORY_PER_CPU` (sent to SBATCH --mem and --mem_per_cpu respectively)

Reply:
Correct, this got lost in my edits. Fixing

Reply:
nice